compiler-rt/lib/scudo/standalone/string_utils.cpp

```diff
@@ -229,6 +229,15 @@ void ScopedString::append(const char *Format, ...) {
   va_end(Args);
 }
 
+size_t ScopedString::copyToBuffer(char *OutputBase, size_t OutputLength) {
+  if (OutputBase && OutputLength) {
```

Comment:
In any case we may need the null check here? If possible, I tend to make it always non-null. (a DCHECK if needed)

Reply:
Done. Thanks.

Reply:
I see. I think that needs to be fixed as well. Let's do the non-null check and returning number of  bytes copied here first